### DISCUSSION TITLE
使用 regex-vis.com 可视化正则表达式

### DISCUSSION THREAD

**Discussion by giscus** [Bowen7/Blog#67] [Announcements]
# 使用 regex-vis.com 可视化正则表达式

https://bowencodes.com/post/regex-vis

**Comment by ChrisSong1994** [comment]
看起来非常好，比 Regulex 要更好用，后续会封装成一个前端库的打算吗？

**Reply by Bowen7** [reply]
有这个打算，等我被裁了就做哈哈
